src/main/java/org/openrewrite/github/ActionsSetupJavaAdoptOpenJDKToTemurin.java

```diff
@@ -26,6 +26,9 @@
 import java.util.Set;
 
 public class ActionsSetupJavaAdoptOpenJDKToTemurin extends Recipe {
+
+    static final JsonPathMatcher DISTRIBUTION_MATCHER = new JsonPathMatcher("..steps[?(@.uses =~ 'actions/setup-java@v[23].*')].with.distribution");
```

Comment:
Is it OK to share this type of state among recipes or should this be declared separately?

Reply:
Yes within a package I think that's fine; would be more work to update otherwise, and as long as it's covered by tests we should be good. Thanks for highlighting it in the changes!